src/components/Episodes/Episodes.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import uuidv1 from 'uuid/v1';
+import { increment } from '../../redux/actions';
+import { getAnimeEpisodes } from '../../redux/thunk';
+
+
+const mapDispatchToProps = (dispatch) => ({
```

Comment:
everywhere you using arrow-functions, feel free to ignore rounded brackets. Et's ES6, so lets use all advantages.